.circleci/config.yml

```diff
@@ -10,7 +10,10 @@ workflows:
       - build:
           matrix:
             parameters:
-              node-version: ["18.12.0", "20.8.1", "21.1.0", "22.0.0"]
+              node-version:
+                - "18" # EOL 30 Apr 2025
+                - "20" # EOL 30 Apr 2026
+                - "22" # EOL 30 Apr 2027
```

Comment:
This orb uses `nvm` under the hood and will pull the most recent in the major range:

![Screenshot 2024-09-04 at 17 44 06](https://github.com/user-attachments/assets/d3d8b2f0-2b58-4ea6-8393-4636b307b026)